crates/testing/resources/eip3074/src/GasSponsorInvoker.sol

```diff
@@ -18,13 +17,14 @@ contract GasSponsorInvoker is BaseAuth {
     /// @return success True if the call was successful
     function sponsorCall(
         address authority,
-        bytes32 commit,
         uint8 v,
         bytes32 r,
         bytes32 s,
         address to,
         bytes calldata data
     ) external pure returns (bool success) {
```

Comment:
I think the function should not be pure given that `authcall` can modify the state, right?